packages/Webkul/Admin/src/Resources/lang/es/app.php

```diff
@@ -3076,18 +3076,18 @@
                 'product-carousel-description'  => 'Muestra productos de forma elegante con un carrusel de productos dinámico y receptivo.',
                 'save-btn'                      => 'Guardar',
                 'select'                        => 'Seleccionar',
-                'slider'                        => 'Deslizador',
-                'slider-add-btn'                => 'Agregar Deslizador',
-                'slider-description'            => 'Personalización de temas relacionada con el deslizador.',
-                'slider-image'                  => 'Imagen del Deslizador',
-                'slider-required'               => 'El campo de deslizador es obligatorio.',
+                'slider'                        => 'Slider',
+                'slider-add-btn'                => 'Agregar Slider',
+                'slider-description'            => 'Personalización de temas relacionada con el slider.',
+                'slider-image'                  => 'Imagen del Slider',
+                'slider-required'               => 'El campo de slider es obligatorio.',
```

Comment:
No need to change this, it is already correct.

Reply:
I think the term used previously to refer to sliders was too ambiguous.